src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/RectMask2D/RectMask2DService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    internal class RectMask2DService : ComponentBroadcasterService<RectMask2DService, RectMask2DObserver>
+    {
+        public static readonly ShortID ID = new ShortID("RMK");
```

Comment:
>RMK [](start = 57, length = 3)

Do we have any mechanism for ensuring that no collisions occur in ids?

Reply:
At runtime you'll get an exception about a key already being added to a dictionary. I could make that a more-explicit error though

Reply:
definitely not required in this review, just was curious :)